### PR TITLE
ref(lw-deletes): enforce ratelimiter

### PR DESCRIPTION
**Pull request by MeredithAnya**
Realized that the allocation policy was active but not enforced, so making it enforced here.

### CODE REVIEW

File: snuba/datasets/configuration/issues/storages/search_issues.yaml [hunk 1]
```diff
@@ -124,7 +124,7 @@ query_processors:
 
 deletion_settings:
   is_enabled: 1
-  max_rows_to_delete: 100000
+  max_rows_to_delete: 2000000
```
Comment (onkar) [inline snuba/datasets/configuration/issues/storages/search_issues.yaml:127]: Is this change intentional? Are we saying that since rate limiter is enforced, we are increasing the default no. of rows to be deleted for search issues?

**Review by onkar — APPROVED** [review]

**Review by kylemumma — APPROVED** [review]

File: snuba/datasets/configuration/issues/storages/search_issues.yaml (hunk 1 above)
Comment (MeredithAnya) [reply snuba/datasets/configuration/issues/storages/search_issues.yaml:127]: this is intentional, it's to match the value passed in ops https://github.com/getsentry/ops/blob/27d30b56d219c1655d2fac9361f0d0a6eb4c708c/k8s/services/snuba/_values.yaml#L558
